src/test/java/entities/HandTest.java

```diff
@@ -0,0 +1,79 @@
+package entities;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class HandTest {
+
+    Hand h1;
```

Comment:
Add access modifiers to class attributes and methods

Reply:
fixed